docs/topics/http/file-uploads.txt

```diff
@@ -122,6 +122,58 @@ field in the model::
             form = UploadFileForm()
         return render(request, 'upload.html', {'form': form})
 
+Uploading multiple files
+------------------------
+
+If you want to upload multiple files at the same time, you need to set the
+``multiple`` boolean attribute of the input element in your
+:class:`~django.forms.Form` first:
+
+.. snippet::
+    :filename: forms.py
+
+    from django import forms
+    from .models import ModelWithFileField
+
+    class FileFieldModelForm(forms.Form):
+        file_field = forms.FileField(widget=forms.ClearableFileInput(attrs={'multiple': True}))
+
+.. note::
+
+    Since we need to set the ``multiple`` boolean attribute of the input
```

Comment:
I think this isn't really needed since there aren't any built-in model fields that can natively handle multiple files, are there?